RecoTracker/LSTCore/standalone/code/core/trkCore.cc

```diff
@@ -437,6 +437,242 @@ std::vector<int> matchedSimTrkIdxs(std::vector<unsigned int> hitidxs,
   return matched_sim_trk_idxs;
 }
 
+//___________________________________________________________________________________________________________________________________________________________________________________________
+std::tuple<std::vector<int>, std::vector<float>> matchedSimTrkIdxsAndFracs(std::vector<unsigned int> hitidxs, std::vector<unsigned int> hittypes, bool verbose, float matchfrac)
```

Comment:
Is a lot of this code just copy-paste from matchedSimTrkIdxs in the same file? Why the need for a new function

Reply:
We want to get the match fractions in addition to the indices. We are not using it now but will need it later on.

Reply:
Can you modify the original ```matchedSimTrkIdxs``` function to add that feature? Or some other solution that gets rid of the copy-paste.

Reply:
I've rewritten `matchedSimTrkIdxs` so that the code is not duplicated